packages/backend/src/clients/Slack/SlackOptions.ts

```diff
@@ -1,15 +1,16 @@
+import { slackRequiredScopes } from '@lightdash/common';
 import { lightdashConfig } from '../../config/lightdashConfig';
 
 export const slackOptions = {
     signingSecret: lightdashConfig.slack?.signingSecret || '',
     clientId: lightdashConfig.slack?.clientId || '',
     clientSecret: lightdashConfig.slack?.clientSecret || '',
     stateSecret: lightdashConfig.slack?.stateSecret,
-    scopes: ['links:read', 'links:write'],
+    scopes: slackRequiredScopes,
 
     // Slack only allow https on redirections
     // When testing locally on http://localhost:3000, replace again https:// with http:// after redirection happens
-    redirectUri: `${lightdashConfig.siteUrl.replace(
+    redirectUri: `${(lightdashConfig?.siteUrl || '').replace(
```

Comment:
should this fail if `siteUrl` is undefined?

Reply:
I don't think we need to throw an error here, worst thing can happen is that the redirect url for slack is not whitelisted